tools/wasme/cli/docs/content/tutorial_code/deploy_tutorials/deploying_with_gloo.md

```diff
@@ -140,14 +140,14 @@ wasme deploy gloo webassemblyhub.io/ilackarms/add-header:v0.1 --id=myfilter --co
 
 This filter adds the header `hello: <value>` to responses, where `<value>` is the value of the `--config` string.
 
-The deployment should have added our filter to the Gloo Gateway. Let's check this with `kubectl`:
+The deployment should have added our filter to the Gloo Edge Gateway. Let's check this with `kubectl`:
 
 ```bash
 kubectl get gateway -n gloo-system '-ojsonpath={.items[0].spec.httpGateway.options.wasm}'
 ```
 
 ```
-map[config:world image:webassemblyhub.io/ilackarms/add-header:v0.1 name:myfilter rootId:add_header_root_id]
+{"filters":[{"image":"webassemblyhub.io/ilackarms/add-header:v0.1","name":"add-header","rootId":"add_header"}]}
```

Comment:
Do we know if this image `webassemblyhub.io/ilackarms/add-header:v0.1` works with Gloo Edge Enterprise 1.6.2? I believe it's pretty old, so it may be incompatible. Somebody following this guide with 1.6.2 using this image might run into issues.

Reply:
I just tested the `ilackarms` image and you're right that it no longer works.  I confirmed that the one I just published to `webassemblyhub.io/jameshbarton/add-header:v0.1` works fine.  I can make that change in the docs.

Reply:
That would be great, thanks! I'd imagine most users walking through this will be using their own filters anyway, but if somebody is walking through the deployment guide and using our example, might as well give them something that works with the latest!